### PR TITLE
[12.0][MIG] delivery_price_rule_untaxed

### PR DESCRIPTION
**Pull request by GSLabIt**
Superseed #203

### CODE REVIEW

**Comment by GSLabIt** [comment]
@rousseldenis @hurrinico @SimoRubi, could u please review?

File: delivery_price_rule_untaxed/tests/test_delivery_price_rule_untaxed.py [hunk 1]
```diff
@@ -53,5 +53,5 @@ def test_untaxed(self):
 
         self.assertEqual(len(line), 1,
                          "Delivery cost is not Added")
-        self.assertEqual(line.price_subtotal, 2 * 300,
+        self.assertEqual(line.price_unit, 2 * 300,
```
Comment (rousseldenis) [inline delivery_price_rule_untaxed/tests/test_delivery_price_rule_untaxed.py]: Why this change ?

Comment (GSLabIt) [reply delivery_price_rule_untaxed/tests/test_delivery_price_rule_untaxed.py]: Honestly i only opened pr and squashed commits, i'll revert this change if needed

Comment (GSLabIt) [reply delivery_price_rule_untaxed/tests/test_delivery_price_rule_untaxed.py]: @rousseldenis reverted, could u please double check and review again?

**Review by SimoRubi — APPROVED** [review]
Thanks for the PR!
Code review: looks good

**Comment by GSLabIt** [comment]
@rousseldenis could you please review and merge?

**Comment by GSLabIt** [comment]
@rousseldenis any chance to have this merged?

**Review by rousseldenis — APPROVED** [review]
